### ISSUE TITLE
tmp.mount support

### ISSUE THREAD

**Issue by erpadmin** [ansible-lockdown/RHEL8-CIS#110]
code is missing deploy custom tmp.mount and notify tmp.mount handler for when 'rhel8cis_tmp_svc: true' in Section 1 vars

**Comment by erpadmin** [comment]
looks like the OS is providing a seeded tmp.mount which doesn't provide the noexec mount option.  the custom tmp.mount will still need to be deployed

```
[root@rhel8-dev2 system]# cat /usr/lib/systemd/system/tmp.mount

#  SPDX-License-Identifier: LGPL-2.1+
#
#  This file is part of systemd.
#
#  systemd is free software; you can redistribute it and/or modify it
#  under the terms of the GNU Lesser General Public License as published by
#  the Free Software Foundation; either version 2.1 of the License, or
#  (at your option) any later version.

[Unit]
Description=Temporary Directory (/tmp)
Documentation=man:hier(7)
Documentation=https://www.freedesktop.org/wiki/Software/systemd/APIFileSystems
ConditionPathIsSymbolicLink=!/tmp
DefaultDependencies=no
Conflicts=umount.target
Before=local-fs.target umount.target
After=swap.target

[Mount]
What=tmpfs
Where=/tmp
Type=tmpfs
Options=mode=1777,strictatime,nosuid,nodev

# Make 'systemctl enable tmp.mount' work:
[Install]
WantedBy=local-fs.target
```

**Comment by uk-bolly** [comment]
Hi @erpadmin 

Thank you for highlighting this issue. I have created a new branch https://github.com/ansible-lockdown/RHEL8-CIS/tree/tmp_mount_%23110 for this fix.
If possible could you check if this works with your setup and i will raise the PR.

Thanks once again

uk-bolly

**Comment by georgenalen** [comment]
This issue was resolved in release 1.4.0
